lib/doh.c

```diff
@@ -182,6 +186,33 @@ doh_write_cb(const void *contents, size_t size, size_t nmemb, void *userp)
   return realsize;
 }
 
+#ifdef USE_HTTPSRR
+static void local_print_buf(struct Curl_easy *data,
+                            unsigned char *buf, size_t len)
+{
+  size_t j;
+  char string_buffer[2048], *ptr;
+
+  ptr = string_buffer;
+  if(len > 1000) {
+    infof(data, "\t:buffer too long %ld", len);
```

Comment:
I don't think the tab should be there. We don't use tabs anywhere else in infof() calls. This goes for the other infof() calls as well.

Since this is more of an internal error than anything else, does it really need an error message at all?

Reply:
was helpful for me debugging to see the RR and ECHConfig values, but got rid of the tab and see above also